Zeeguu Reader/Zeeguu Reader/ArticleViewController.swift

```diff
@@ -320,13 +320,18 @@ class ArticleViewController: UIViewController, WKNavigationDelegate, WKScriptMes
 			action.setBookmarkID(b)
 			
 			if (self.pronounceTranslatedWord) {
+				let _ = try? AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback)
+				let _ = try? AVAudioSession.sharedInstance().setActive(true)
+				
 				let synthesizer = AVSpeechSynthesizer()
 				
 				let utterance = AVSpeechUtterance(string: word)
 				utterance.voice = AVSpeechSynthesisVoice(language: self.article?.feed.language)
 				
 				synthesizer.stopSpeakingAtBoundary(AVSpeechBoundary.Immediate)
 				synthesizer.speakUtterance(utterance)
+				
+				let _ = try? AVAudioSession.sharedInstance().setActive(false)
```

Comment:
I wanted to avoid the try catch stuff. Mainly because I really am not interested in the case where this fails. The `?` in `try?` makes the result an optional `Void` (type being `Void?`). Because it is not returning `Void` anymore, it gives a warning about unused result, so I used the `let _` syntax. Although, I just realized that the `let` here is not needed, so it becomes `_ = try? ...`.

See also [this](http://stackoverflow.com/a/33621334) answer.